### ISSUE TITLE
Update request package - current version 2.34.0 vulnerable

### ISSUE THREAD

**Issue by odnarb** [balderdashy/sails.io.js#58]
Not sure if this is necessary to include an issue, but just trying to follow some sort of protocol.

Please update the request package dependency to >=2.40.0

per request's dependency on qs:
https://github.com/request/request/pull/992
https://github.com/request/request/pull/993

Also see on nodesecurity.io:
https://nodesecurity.io/advisories/qs_dos_extended_event_loop_blocking
https://nodesecurity.io/advisories/qs_dos_memory_exhaustion

**Comment by sailsbot** [comment]
Thanks for posting, @odnarb.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
